.github/workflows/pytest-gpu.yml

```diff
@@ -0,0 +1,64 @@
+# Adapted from https://github.com/weeyin83/vm-actions
+name: Deploy VM on Azure and run GPU tests
+
+env:
+  OUTPUT_PATH: ${{ github.workspace }}
+
+on: [push]
+
+jobs:
+
+  # Deploy VM in Azure
+  DeployVM:
+    runs-on: ubuntu-latest
+
+    steps:
+    # checkout code from repo
+    - name: checkout repo
+      uses: actions/checkout@v1
+
+    - name: extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_branch
+
+    - name: look for ps1 file
+      run: |
+        ls '${{ env.OUTPUT_PATH }}/PSA'
+    - name: deploy VM
```

Comment:
nitpicking: blank spce before this line?

Reply:
Added.